frontend/express/public/core/events/templates/overview.html

```diff
@@ -0,0 +1,144 @@
+<script type="text/x-template" id="events-overview">
+    <div v-bind:class="[componentId]">
+        <cly-header>
+            <template v-slot:header-left>
+                <h2> {{i18n('events.overview.title.new')}} 
+                </h2>
+                <span class="el-icon-info" style="margin-left:10px" v-tooltip.top-center="description">
+                </span>
+            </template>
+        </cly-header>
+		<div class="bu-columns bu-is-gapless bu-is-mobile bu-is-multiline">
+			<events-breakdown-horizontal-tile
+				:trend="item.trend"
+				:change="item.change"
+				:key="idx"
+				v-for="(item, idx) in eventsOverview">
+				<h4 slot="title" class="text-uppercase cly-events-breakdown-horizontal-tile__titleoverview">{{item.name}}</h4>
+				<div slot="countValue" class="cly-events-breakdown-horizontal-tile__overviewvalue">{{item.value}}</div>
+			</events-breakdown-horizontal-tile>
+		</div>
+		<h4 class="cly-vue-events-overview-top-events">{{i18n('events.overview.top.events.by.count')}} </h4>
+		<div class="bu-columns bu-is-gapless bu-is-mobile bu-is-multiline">
+			<events-breakdown-horizontal-tile 
+				:trend="item.trend"
+				:change="item.change"
+				:key="idx"
+				v-for="(item, idx) in topEvents">
+				<h4 slot="title" class="text-uppercase cly-events-breakdown-horizontal-tile__title">{{item.name}}</h4>
+				<div slot="countValue" class="cly-events-breakdown-horizontal-tile__value">{{item.value}}</div>
+				<div class="bu-level-right" slot="totalPercentage">
+					<div class="bu-level-item cly-events-breakdown-horizontal-tile__percentage">
+					 {{item.percentage}}{{i18n("events.percentage.of.total")}}
+					</div>
+				</div>
+				<div slot="progressBar">
+					<progress class="bu-progress bu-is-link" :value="item.percentage" max="100">{{item.percentage}}%</progress>
+				</div>
+			</events-breakdown-horizontal-tile>
+		</div>
+        <cly-main>
+            <h3>{{i18n('events.overview.metrics')}} </h3>
+            <detail-tables></detail-tables>
+        </cly-main>
+        <div>
+            <h3 style="padding-left:24px">{{i18n('events.overview.monitor')}} </h3>
+            <div style="padding-left:24px">{{i18n('events.overview.time.period')}}
+            <cly-date-picker-g class="cly-vue-events-overview-date-picker"></cly-date-picker-g>
+		</div>   
+		</div>
+        </div>
+    </div>
+</script>
+<script type="text/x-template" id="overview-tables">
```

Comment:
are you using this template?
couldn't find it in the code.

Reply:
It was in initial requirement where we wanted to show 2 tables on overview page . But that has changed now . Will remove:)